packages/showcases/svelte/stories/components/structure/VtmnDivider/VtmnDivider.stories.svelte

```diff
@@ -27,6 +27,7 @@
 <Story class="block" name="Overview" let:args>
   <VtmnDivider
     class="story-divider-horizontal story-divider-vertical"
+    aria-labelledby="title-header-0"
```

Comment:
And where is the element with the id `#title-header-0`? If you add this, you need to add this element in order to label the divider.

Reply:
Replace "aria-labelled-by" with "labelId"

Reply:
I'm on it :)